apps/microsoft-teams/frontend/src/hooks/useGetTeamsChannels.ts

```diff
@@ -8,7 +8,6 @@ const useGetTeamsChannels = () => {
   const [error, setError] = useState<Error>();
   const [loading, setLoading] = useState<boolean>(true);
   const sdk = useSDK<ConfigAppSDK>();
-  const { tenantId } = sdk.parameters.installation as AppInstallationParameters;
```

Comment:
unrelated to this PR, but this is an unnecessary dependency.  `tenantId` is only used in the `useCallback()` change deps.  See line 46